src/axolotl/integrations/diffusion_gemma/collator.py

```
@@ -0,0 +1,155 @@
+        tokenizer,
+        canvas_length: int = 256,
+        block_selection: str = "random",
+        pad_to_multiple_of: int | None = None,
+        seed: int | None = None,
+        **_ignored,
+    ):
+        self.tokenizer = tokenizer
+        self.canvas_length = canvas_length
+        self.block_selection = block_selection
+        self.pad_to_multiple_of = pad_to_multiple_of
+        self._rng = random.Random(seed)
```

Ruff (0.15.15)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)